code/modules/projectiles/guns/energy/special.dm

```diff
@@ -289,6 +294,48 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 	origin_tech = null
 	m_amt = 0
 
+/obj/item/weapon/gun/energy/plasmacutter
+	name = "plasma cutter"
+	desc = "A mining tool capable of expelling concentrated plasma bursts. You could use it to cut limbs off of xenos! Or, you know, mine stuff."
+	icon_state = "plasmacutter"
+	item_state = "plasmacutter"
+	modifystate = "plasmacutter"
+	origin_tech = "combat=1;materials=3;magnets=2;plasmatech=2;engineering=1"
+	projectile_type = /obj/item/projectile/plasma
+	flags = CONDUCT | OPENCONTAINER
+	attack_verb = list("attacked", "slashed", "cut", "sliced")
+	charge_cost = 250
+	fire_delay = 10
+	icon_override = 'icons/mob/in-hand/guns.dmi'
+	can_charge = 0

```

Comment:
Made the fire the laser sound for consistency with the mech version, especially considering how the pulse sound usually means "OH FUCK THERE'S A DEATHSQUAD" (or adminboos).